katas/langtonant/bryan/bryan_Lant.test.js

```diff
@@ -0,0 +1,22 @@
+import {makeGrid} from './bryan_Lant';
+
+
+test('makeGrid function should display new clean height x width grid filled with -', () => {
+    const gridTest = makeGrid(3, 3);
+    expect(gridTest).toStrictEqual([['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']]);
+});
+
+
+import {cellState} from './bryan_Lant';
+test('cellState returns the element in the coordinates given', () => {
+    const gridTest = [['1', '2', '3'], ['4', '5', '6'], ['7', '8', '9']];
+    expect(cellState(gridTest, 0, 2)).toBe('3');
+});
+
+import {direction} from './bryan_Lant';
```

Comment:
it's more common to group all imports at the top of the file, please update it so it follows this suggestion